core/monitor/auction.go

```diff
@@ -252,26 +239,10 @@ func (a AuctionState) IsOpeningAuction() bool {
 	return a.trigger == types.AuctionTriggerOpening
 }
 
-func (a AuctionState) IsLiquidityAuction() bool {
-	// FIXME(jeremy): the second part of the condition is to support
-	// the compatibility on 72 > 73 snapshots.
-
-	return a.trigger == types.AuctionTriggerLiquidityTargetNotMet ||
-		a.trigger == types.AuctionTriggerUnableToDeployLPOrders
-}
```

Comment:
I'm pretty sure we still need at least the first part to make sure we are compatible with market being in liquidity auction before an upgrade?

Or we could also just make sure that at the time of the migration, the liquidity auction type is migrated to something else.

Reply:
Updated, this should be fine now